brianlong/www.validators.app
# 186685548 skipped after chart

#### What's this PR do?
- add skipped after chart for validators

#### How should this be manually tested?
- make sure you have some skipped_after_history in scores by running gather and score daemons
- check validator index, stake pools and dc details and check if the charts are valid

#### What are the relevant tickets?
- [URL to the PivotalTracker ticket](https://www.pivotaltracker.com/story/show/186685548)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
